SimPEG/electromagnetics/static/resistivity/receivers.py

```diff
@@ -185,7 +185,7 @@ def eval(self, src, mesh, f):  # noqa: A003
             try:
                 if mesh.dim == 2:
                     return v / self.geometric_factor[src][:, None]
-                return v / self.geometric_factor[src]
+                return v[:,0] / self.geometric_factor[src]
```

Comment:
Would you mind changing the fix to:
```suggestion
                return v / self.geometric_factor[src][:, None]
```
So that the returned array has a consistent number of dimensions?

Reply:
Done!